src/scripts/main.js

```diff
@@ -1,7 +1,25 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageBlock = document.createElement('div');
+  const messageTitle = document.createElement('h2');
+  const messageContent = document.createElement('p');
+
+  messageBlock.className = `notification ${type}`;
+  messageBlock.style.top = `${posTop}px`;
+  messageBlock.style.right = `${posRight}px`;
+  messageTitle.className = 'title';
+
+  messageTitle.textContent = title;
+  messageContent.textContent = description;
+
+  messageBlock.append(messageTitle);
+  messageBlock.append(messageContent);
+  document.querySelector('body').append(messageBlock);
+
+  setTimeout(() => {
+    messageBlock.hidden = true;
+  }, 3000);
```

Comment:
in task requirement 2 seconds 

pushNotification creates an element to display the message, appends it to the document, and removes it from the DOM after 2 seconds.